_posts/2017/2017-09-12-five-compelling-reasons-why-you-should-care-about-terminology.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "5 compelling reasons why you should manage your terminology"
+date: "2017-09-12 09:30:00"
+image: blog-header/jellybears.jpg
+categories: tech-stories
+authors: ["Birgit"]
+---
+
+Communicating in a clear and consistent manner, and saving money at the same time.
+Sounds great?
+Let's tackle it!
+
+This one is about terminology work.
+And the importance to implement it in your company, your processes, your daily work.
+The **Who** and **How** and **Where** depend on every company's specific conditions and requirements.
+So better everyone figures that out for themselves.
+I'd like to lose a few words about **Why** terminology work matters:
+
+## Do you know what's meant?
+
+Terminology work has a very clear purpose: being consistent in your language and writing.
+Whether it is about developing a product, publishing press releases, or communicating within the company: talking about business issues in the same words, and using the same writing makes everyone's life easier and, primarily, more efficient.
+And productive.
+And satisfying.
+Everybody understands what is meant by using the same term, with a meaningful definition.
+It's as simple as that.
+
+## Go tell it on the mountain
+
+Once there is a consensus about the right terms to be used within a department, for a project, throughout the company, or whatever else: spread the word.
+Make an Excel sheet (did I really write that down?!), manage a list on your intranet, use one of these super professional terminology management tools (my absolute favorite!), or post messages on a notice board, but - please - communicate it.
+To everyone involved.
+And could be involved.
+And should be involved.
+And would like to be involved.
+Sounds like the whole company, right?
+
+Consistency amongst all departments saves everyone a lot of time and money, and, if you live it, demonstrates professionalism to your customers (see Customers in love).
+
+## Sharing is caring
+
+This being said, I get to the point of knowledge sharing.
+You've your terminology recorded.
+It's somewhere.
+Not just in your heads.
+
+A new colleague joins the team.
+Works on your project.
+Comes along with their own magic phrasing.
+And you can just raise your hand, open a sheet (or a website), and put them in their place.
+Or, what if this colleague doesn't know what's `this` or `that`?
+Simply refer to your terminology.
+And things will become clearer.
+
+## Customers in love
+
+Now that you have your terminology on board, everybody knows about it (even your new colleagues {% emoji wink %}) and uses it consistently, you've already made a big step forward.
+This way you can produce high-quality content
+Great texts that are easy to read and understand.
+Texts that your customers will love, because they're consistent, thus user-friendly.
+And it brands your company - positively.
+It shows that you value language and consistent communication.
+And - hey - we're talking again about professionalism - wow!
```

Comment:
By skipping the professionalism comment earlier above, you could mention it in this place a little bit less "ketzerisch"?;)